drivers/hmis/app/models/hmis/form/processor.rb

```diff
@@ -0,0 +1,196 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Form::Processor < ::GrdaWarehouseBase
```

Comment:
Should this be `AssessmentProcessor` since this isn't generic enough to handle services, events, and any other items, or is the idea that it eventually becomes generic?